cli/lib/cli.js

```diff
@@ -253,6 +256,7 @@ const addCypressRunCommand = (program) => {
   .option('-k, --key <record-key>', text('key'))
   .option('--headed', text('headed'))
   .option('--headless', text('headless'))
+  .option('--no-auto-cancel-after-failures', text('noAutoCancelAfterFailures'))
```

Comment:
See this: https://github.com/tj/commander.js#other-option-types-negatable-boolean-and-booleanvalue

I don't think we need to define this option here

Reply:
Without having it define we get this error

`error: unknown option: --no-auto-cancel-after-failures`

Reply:
Then I'd suggest we just remove it. I know I originally advocated for it, but having it show up twice (whereas the other options we only define it once). Originally based on my reading of the docs I thought we got the `--no-*` flag for free, but since it must be defined I would recommend it's not worth the additional complexity of documentation and weight. Setting `false` already works.